lib/elixir/lib/module/types.ex

```diff
@@ -451,4 +484,7 @@ defmodule Module.Types do
 
   defp integer_type?(:integer), do: true
   defp integer_type?(_other), do: false
+
+  defp call_to_mfa({{:., _, [mod, fun]}, _, args}), do: {mod, fun, length(args)}
+  defp call_to_mfa({fun, _, args}), do: {Kernel, fun, length(args)}
```

Comment:
why is it only `{fun, _, args}` for Kernel? Should we also check `fun` is an atom?

Reply:
> why is it only {fun, _, args} for Kernel?

Because it's the reverse of https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/module/types.ex#L356.

> Should we also check fun is an atom?

We can check but I usually those kind of kinds for internal functions.

Reply:
I would check only to be sure and help us catch bugs if `fun` becomes something else in the future.